src/actions/PetitionsActions.js

```diff
@@ -29,19 +29,26 @@ export function fetchPetitions ({ location, params }, group) {
       .then(response => dispatch(receivePetitions(
         response,
         queryParams,
-        queryString,
-        group
+        queryString
       )));
   };
 }
 
 export function fetchGroupedPetitions (props = {}, groups = []) {
-  const sortQuery = getHomePagePetitionsQuery();
-  const location = Object.assign({}, props.location, sortQuery);
-
   return (dispatch) => {
-    const groupDispatches = groups.map(group =>
-      dispatch(fetchPetitions({ location }, group)));
+    dispatch(requestPetitions());
```

Comment:
Not sure this logic for `isLoading` is re-useable here actually.
We set it to `petitions: { isLoading: true }` here and then after the first group finishes it's set to `false`?

Do you think we need;

```
latest: {
  isLoading
},
trending: {
  isLoading
}
```

so we could display a loading spinner for each group individually?

Not critical, I guess